### PR TITLE
[DO-599] prev/next buttons description customization

### PR DESCRIPTION
**Pull request by d0rich**

### CODE REVIEW

**Comment by StanG4313** [comment]
@d0rich , am I right that our custom descriptions will be limited with 2 lines of text? 
And also, should we create some documentation section/page to describe this feature for our users?

**Comment by d0rich** [comment]
> @d0rich , am I right that our custom descriptions will be limited with 2 lines of text? And also, should we create some documentation section/page to describe this feature for our users?

Only descriptions in prev/next cards will be limited, if you specified `display: 'limited'`. By default it will remain full.

I think I will add some documentation.

**Review by StanG4313 — CHANGES_REQUESTED** [review]
It will be good after adding some documentation of this feature

File: docs/content/3.customization.md [hunk 1]
```diff
@@ -21,6 +21,45 @@ Set up your very own logo with an image of your choice in two ways:
...
+    prevNextCards: {
+      /**
+       * Display options for the description of the page in prev/next cards
+       */
+      description: {
+        /**
+         * Choose display type for the description of the page in prev/next cards.
+         * 
+         * 'full' - display full description
+         * 'limit' - display first n lines of description
```
Comment (StanG4313) [inline docs/content/3.customization.md:44]:
```suggestion
         * 'limited' - display first n lines of description
```

With 'limit' it didn't changed

**Review by StanG4313 — APPROVED** [review]

**Pull request merged**